cluster-up/cluster/kind/common.sh

```diff
@@ -152,15 +174,9 @@ function setup_kind() {
         docker exec $node /bin/sh -c "curl -L https://github.com/containernetworking/plugins/releases/download/v0.8.5/cni-plugins-linux-amd64-v0.8.5.tgz | tar xz -C /opt/cni/bin"
     done
 
-    echo "ipv6 cni: $IPV6_CNI"
```

Comment:
not related to this PR imo

Reply:
please see
https://github.com/kubevirt/kubevirtci/pull/463

i removed the whole provider there, but didnt touch this code since you already remove it,
again i think its not belong to this PR context imho

Reply:
Hey you can remove it anyway and if your PR will merge first I will change mine

Reply:
sure will do thx